index.css

```diff
@@ -951,6 +1069,35 @@ header li a:hover {
     flex-direction: column;
   }
 
+  #popUp {
+    width: 80%;
+    top: 50%;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  #popUp img,
+  #popUp .img,
+  #popUp.active .text {
+    width: 100%;
+  }
+
+  #popUp.active .text ul li {
+    margin-right: -21px;
+  }
+
+  #popUp.active .text ul li a {
+    font-size: 12px;
+    padding-right: 2px;
+  }
+
+  #popUp.active .text ul li a img {
+    width: 20px;
+    padding-right: 15px;
+  }
+
```

Comment:
- [ ] As mentioned by the previous reviewer please make sure your design matches the Figma design. look closely at the `font-color`s, `font-family`, especially in the badges

|Your Design|Figma|
:-|-:
|![image](https://user-images.githubusercontent.com/18730106/151248371-d157f5e7-5600-4806-af05-4a907ffc1793.png)|![image](https://user-images.githubusercontent.com/18730106/151248459-6aa13053-a974-4deb-b7ce-b98f3f133ff0.png)|

|Your Design|Figma Design|
-|-
![image](https://user-images.githubusercontent.com/18730106/151248665-71eb3d88-fd79-4360-83f4-639708ce4d5c.png)|![image](https://user-images.githubusercontent.com/18730106/151248730-ef1f1397-dfe3-44a3-b779-478e7747774a.png)